site/content/docs/main/guides/fips.md

```diff
@@ -101,9 +101,11 @@ Note that if you do build with FIPS mode outside of the build container, you can
 We can first compile the Envoy binary by running the following in a `bash` shell from the Envoy source directory:
 
 ```bash
-BAZEL_BUILD_EXTRA_OPTIONS="--define boringssl=fips" ENVOY_DOCKER_BUILD_DIR=<envoy-output-dir> ./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.release.server_only'
+BAZEL_BUILD_EXTRA_OPTIONS="--define boringssl=fips" ENVOY_DOCKER_BUILD_DIR=<envoy-output-dir> ./ci/run_envoy_docker.sh './ci/do_ci.sh bazel.release //test/exe:envoy_static_test'
 ```
 
+*This command mimics the Envoy release CI process with the target `bazel.release` but differs in only running a single test for brevity. You may omit the `//test/exe:envoy_static_test` test entirely to run the full suite of Envoy tests.*
+
 Replace `<envoy-output-dir>` with a directory you would like the build output to be placed on your host computer.
 Once that build completes, you should have a file named `envoy_binary.tar.gz` in your specified output directory.
```

Comment:
`envoy_binary.tar.gz` is not created. 
`<envoy-output-dir>/envoy/x64/bin/release.tar.zst` is created instead and needs to be copied to the <envoy-repo-dir>/linux/amd64/  directory. After this we can run the `docker build -f ./ci/Dockerfile-envoy .`  command as specified in the existing documentation.

Reply:
Just to clarify, we don't need to extract `release.tar.zst` the `[Dockerfile is expecting that file](https://github.com/envoyproxy/envoy/blob/af8aef48b5395d32d4bbac5aa3ed0a84d3bf31d1/ci/Dockerfile-envoy#L12)`.

Reply:
ah thats right, forgot to update the copy instructions, will add that as well 👍🏽

Reply:
updated the copy instructions, please take another look!

Reply:
Thanks @sunjayBhatia! 
I got it working, however one thing that tripped me up is that I was running an older version of docker and the docker build steps failed.  Took me ages to figure out that the docker version had anything to do with the problem. Do you think it's worth adding a note to suggest the build is carried out with the latest version of docker?

Reply:
oh man, I just noticed there is a new note there. All good from me then